cinema/views.py

```diff
@@ -43,11 +88,38 @@
 
         return MovieSerializer
 
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        return Response(response.data["results"])
```

Comment:
In `MovieViewSet`, the `list` method override removes pagination metadata. Consider whether this is intended or if pagination should be preserved.